app/model/editions/EditionsCollection.scala

```diff
@@ -0,0 +1,62 @@
+package model.editions
+
+import java.time.ZonedDateTime
+import play.api.libs.json.Json
+
+import scalikejdbc.WrappedResultSet
+
+case class EditionsCollection(
+    id: String,
+    name: String,
+    prefill: String,
+    is_hidden: Boolean,
```

Comment:
Tiny one but I'm just looking at this and is there any reason this isn't `camelCase`? 🐫

Reply:
Sorry, typo